pettingzoo/utils/wrappers/capture_stdout.py

```diff
@@ -1,23 +1,25 @@
 from pettingzoo.utils.capture_stdout import capture_stdout
+from pettingzoo.utils.env import AECEnv
 from pettingzoo.utils.wrappers.base import BaseWrapper
 
 
 class CaptureStdoutWrapper(BaseWrapper):
     """Takes an environment which prints to terminal, and gives it an `ansi` render mode where it captures the terminal output and returns it as a string instead."""
 
-    def __init__(self, env):
+    def __init__(self, env: AECEnv):
+        assert hasattr(env, "render_mode"), f"Environment {env} has no render_mode."
         assert (
-            env.render_mode == "human"
-        ), f"CaptureStdoutWrapper works only with human rendering mode, but found {env.render_mode} instead."
+            env.render_mode == "human"  # pyright: ignore[reportGeneralTypeIssues]
```

Comment:
this first one isn’t needed, the only time when pyright comments like this will be read is at the end of the parentheses (line 13)

Reply:
Nvm checked and it is requited